containers/Dockerfile.al2

```diff
@@ -1,7 +1,7 @@
 # Copyright 2020-2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-FROM amazonlinux:2 as builder
+FROM public.ecr.aws/amazonlinux/amazonlinux:2 as builder
```

Comment:
Can you explain in the commit message why are we doing this change?

Reply:
nit: is it possible to define a constant for `public.ecr.aws/amazonlinux/amazonlinux:2` so that we don't need to change it in multiple places?